_delphi_utils_python/delphi_utils/weekday.py

```diff
@@ -12,11 +12,13 @@ class Weekday:
     """Class to handle weekday effects."""
 
     @staticmethod
-    def get_params(data, denominator_col, numerator_cols, date_col, scales, logger):
+    def get_params(data, denominator_col, numerator_cols, date_col, scales, logger, solver=cp.CLARABEL):
         r"""Fit weekday correction for each col in numerator_cols.
 
         Return a matrix of parameters: the entire vector of betas, for each time
         series column in the data.
+
+        solver: Historically use "ECOS" but due to numerical issues, "CLARABEL" is now default.
         """
         tmp = data.reset_index()
```

Comment:
this puts the default into the body of the method, and it clarifies version constraints (see the [cvxpy update notes](https://www.cvxpy.org/updates/index.html#:~:text=In%201.5%2C%20CVXPY%20plans%20to%20start%20using%20Clarabel%20instead%20of%20ECOS%20by%20default%20for%20some%20categories%20of%20problems.%20In%201.6%2C%20we%20plan%20to%20no%20longer%20install%20ECOS%20as%20a%20CVXPY%20dependency.))
```suggestion

        solver: Historically used "ECOS" but due to numerical stability issues, "CLARABEL" (introduced in cvxpy 1.3) is now the default solver in cvxpy 1.5.
        """
        if solver_override is None:
            solver = cp.CLARABEL
        else:
            solver = solver_override
        tmp = data.reset_index()
```